onnxruntime/python/tools/tensorrt/perf/README.md

```diff
@@ -0,0 +1,160 @@
+# TensorRT Performance Test Script
+This script fetches models from ONNX Model Zoo and runs ONNX Runtime inference using TensorRT and CUDA as execution provider to get TensorRT performance gain compared with CUDA, as well as other metrics such as percentage of model operators and execution time that run on TensorRT.
```

Comment:
do we always want to fetch models from the zoo each time?
the models might already be available locally and we want to point the script at some directory.

Reply:
Script will fetch models from zoo if it's the first time running it. After the perf is done, script calls cleanup_files() to remove all the models, meta data and input/output test data. 
If you delete cleanup_files(), script won't clean up the files and next time it will use the existing files on the local machine.